python/ray/autoscaler/_private/command_runner.py

```diff
@@ -558,9 +589,11 @@ def run_rsync_down(self, source, target, options=None):
             source.lstrip("/"),
         )
         host_mount_location = os.path.dirname(host_source.rstrip("/"))
```

Comment:
Do we need `host_source.rstrip("\\")`?

Reply:
we don't need to strip the '\', the python path libraries handle paths ending in `\` correctly on windows. 
There were cases where paths to windows folders were specified with `/` and then used later to join paths which resulted in invalid paths (like `c:\Users\marosset/.ssh` which was causing issues when the file was getting accessed)